xml/ha_config_cli.xml

```diff
@@ -941,6 +941,11 @@ Membership information
    <step>
     <para>
      Show your changes:
+     <remark>taroth 2018-03-05: ygao, according to 
+     https://wiki.clusterlabs.org/wiki/Pacemaker_2.0_Configuration_Changes,
+     the cluster property 'expected-quorum-votes' used in the screen below
+     no longer exist, but it is not clear with what to replace it - please
+     advise!</remark>
```

Comment:
From SLE12 (corosync-2),  quorum is not determined by pacemaker anymore. Instead, corosync provides quorum service, and there's corresponding "expected_votes" parameter can be configured in /etc/corosync/corosync.conf.

Reply:
Ah, right, we mention the "expected_votes" in section 'Defining the Communication Channels' in the file 'ha_installation.xml'. 
For the time being, I commented this section completely, because it is outdated in more than one aspect meanwhile. Added a comment to define a new, useful example for setting global cluster options with crmsh and to re-add the section for the future.